source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -0,0 +1,310 @@
+===================
+Search Geospatially
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+On this page you will learn the how to search **geospatial data** in the
+MongoDB Java Driver.
+
+Geospatial data is data that represents a geographical location on
+the surface of the Earth. Examples of geospatial data include:
+
+- Locations of movie theaters
+- Borders of countries
+- Routes of bicycle rides
+- Dog exercise areas in New York City
+
+You should read this guide if you would like to:
+
+- Understand the different geospatial data formats in MongoDB.
+- Build geospatial queries.
+
+Coordinates on an Earth-like Sphere
+-----------------------------------
+
+You can represent geospatial data on an Earth-like sphere with **GeoJSON**.
+GeoJSON is a geospatial data format created by the Internet Engineering
+Task Force (IETF). You can use the GeoJSON format to store your data in MongoDB,
+and you must use the GeoJSON format to query your geospatial data in MongoDB. 
```

Comment:
I've given this more thought. The message here is mixed, still, for no good reason. "You can use x to store data, you must use x to query". I can't see any other outcome other than possible confusion over this. Why not spell it out? 

Also, the structure is still somewhat repetitive, and doesn't flow well. The bit about the IETF feels shoehorned in now. Here's an idea to work with.

```suggestion
You can store geospatial data on Earth with **GeoJSON**, a geospatial data format 
[created by the Internet EngineeringTask Force (IETF)(does this really add anything?)].

 MongoDB uses the :manual:`same reference system </reference/glossary/#std-term-WGS84>`
 as GPS satellites to calculate geospatial locations.
```

Reply:
Thanks for the note. I agree that my fist paragraph in this section is too repetitive, and I agree that "can... must" doesn't add anything and that saying this doesn't suggest that legacy coordinate pairs are deprecated (as they are meant to be on a 2d plane rather than earth). 

I rewrote this section to be more task oriented (starting with verbs "store" and "query"), and shortened the sentence that introduces IETF.  

I added the fact about satellites to the examples as a fun fact (as there is a picture of a map with a geometry on it).

Fix: